rskj-core/src/main/java/co/rsk/rpc/json/JsonResponseSizeLimiter.java

```diff
@@ -0,0 +1,106 @@
+package co.rsk.rpc.json;
+
+import co.rsk.rpc.exception.JsonRpcResponseLimitException;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import org.ethereum.rpc.exception.RskJsonRpcRequestException;
+
+import java.util.Iterator;
+import java.util.Map;
+
+public class JsonResponseSizeLimiter {
```

Comment:
I don't like too much the way the symbols sizes are calculated when traversing through the different elements of the JSON object. Any suggestion about how to improve is well received.